data/en/test_dropdown_mandatory.json

```diff
@@ -0,0 +1,45 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.2",
+    "survey_id": "0",
+    "title": "Dropdown Mandatory",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "groups": [{
+        "id": "dropdown",
+        "title": "Dropdown Mandatory",
+        "blocks": [{
+                "type": "Question",
+                "id": "dropdown-mandatory",
+                "questions": [{
+                    "type": "General",
+                    "id": "dropdown-mandatory-question",
+                    "title": "Which football team do your support?",
+                    "answers": [{
+                        "type": "Dropdown",
+                        "id": "dropdown-mandatory-answer",
+                        "mandatory": true,
+                        "options": [{
+                                "label": "Liverpool",
+                                "value": "Liverpool"
```

Comment:
No one is going to choose this value surely ?!?!

Reply:
This is the only value needed!